lpa-submissions-e2e-tests/wait.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require("node-fetch");
+
+console.log('Waiting for ZAP');
+
+const max_wait = 2000;
+
+async function wait(ms) {
+    return new Promise(resolve => {
+        setTimeout(resolve, ms);
+    });
+}
+
+['http://localhost:8080/JSON/pscan/view/recordsToScan'].map(async (url) => {
```

Comment:
looks like a leftover bit of the zap stuff?
as an aside- i'd like to catch up with people about this wiring as i think there are much lighter touches available.. that said i would like to understand what all this is about as it feels like this is trying to wire the scanning into cypress to get it as part of the existing cypress output?

Reply:
I think this is a discussion we need to have between teams for sure. This work was being led by @shilpabasu22 I believe so best to reach out to her